corehq/apps/data_dictionary/templates/data_dictionary/base.html

```diff
@@ -17,10 +17,23 @@ <h2 class="text-hq-nav-header">{% trans "Data Dictionary" %}</h2>
   {% registerurl 'data_dictionary_json' domain %}
   {% registerurl 'update_case_property' domain %}
   {% initial_page_data 'typeChoices' question_types %}
+  {% initial_page_data 'fhirResourceTypes' fhir_resource_types %}
   <div data-bind="saveButton: saveButton"></div>
   <div class="row">
     <div class="col-md-10">
       <h3 data-bind="text: $root.activeCaseType()"></h3>
+      {% if fhir_integration_enabled %}
+        <div class="form-inline" data-bind="visible: fhirResourceTypes().length">
+          {% trans "FHIR Resource Type" %}
+          <select class="form-control"
```

Comment:
select2 would be nice for UI consistency, we're using them more and more places.

Reply:
that would mean adding a class "hqwebapp-select2" and importing `'select2/dist/js/select2.full.min'` or something more?

Reply:
Yep, that class, but import `hqwebapp/js/widgets`, which will save you from needing to call `.select2()` yourself. There are some docs on select2s [here](https://www.commcarehq.org/styleguide/molecules/#molecules-selections), somewhat hidden in the "Select2" panel under the "Code" tab.

Reply:
thanks @orangejenny 
I have made the change https://github.com/dimagi/commcare-hq/pull/29210/commits/8bb095a38bafa956b6ea7fe86ce49becc8c6c555
one thing though, now my select container is stretched how do i keep it small? i tried adding "col-sm-3" but then that lead the following items to come floating up to the right of select.